src/app/api/get-many-quizzes/route.ts

```diff
@@ -1,8 +1,17 @@
-import prisma from '@/lib/db/client'
+import {getManyQuizService, getQuizService} from '@/lib/core/service'
```

Comment:
From our previous conversation, I think we will be using REST methods for the API, eg:

GET /api/quiz/123
POST /api/quiz
PUT /api/quiz/123
DELETE /api/quiz/123

Reference: https://www.restapitutorial.com/lessons/httpmethods.html